### PR TITLE
Cleanup .gitignore file

### PR DESCRIPTION
**Pull request by shulard**
I am targetting this branch, because this is a pedantic branch.

Related to: sonata-project/dev-kit#211

## Subject

Remove plateform specific ignore entries from .gitignore file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,7 @@
-vendor/
-report/*
-Resources/doc/_build/
+/build
+/Resources/doc/_build
+/vendor
+/report/*
```
Comment (greg0ire) [inline .gitignore]: Not sure this is still needed (I see no reference to it in the project). @dbu @lsmith77 ?

Comment (dbu) [inline .gitignore]: would not remember what that is, lets remove the line.

**Comment by shulard** [comment]
I've pushed a new commit with the `report/*` removed.

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,6 @@
-vendor/
-report/*
-Resources/doc/_build/
+/build
+/Resources/doc/_build
+/vendor
 composer.lock
+phpunit.xml
 .php_cs.cache
```
Comment (core23) [inline .gitignore]: Please sort the files as well like in https://github.com/sonata-project/SonataUserBundle/blob/3.x/.gitignore

**Pull request merged**